### DISCUSSION TITLE
How participatory mapping can be better utilized for civil society

### DISCUSSION THREAD

**Discussion by alizezila** [albertkun/21S-ASIAAM-191A#56] [Past Thinking Caps]
Hey all! For this week's thinking cap, I want to reflect on how people can not only use maps but create maps that better reflect their lived environments. As our guest lecturer Mariah Tso pointed out during last weeks class, maps don't necessarily need a neat line or a legend, they exist in many forms all around us at all times. I think if more people had more awareness about what a map CAN be rather than what it is, more work can be done to help better civil society. A great map that can utilized to fight food insecurity and aid LA's homeless community is the [LA Community Fridges map](https://community-fridge.bubbleapps.io/). It's a great source that even shows what the fridge is lacking, so if one needs specific items they know which fridges have what. There are some drawbacks, the fridges require "check-ins" that might not be updated quickly and some fridges are under scrutiny by the city and might negatively affect the unhoused community who want to use those fridges. Though, I think it is a great tool that can help others and work as a guide for other participatory maps, such as mapping which grocery stores/restaurants accept EBT, mapping areas where street vendors can find resources, etc. It is always important to remember that maps are stories and map-making is essentially story-telling and these processes are always imbued with power, who has access to the adequate resources and opportunities to tell these stories.

**Comment by rachan2023** [comment]
I agree! In order to improve participatory mapping and critical cartography, we need to move away from the idea that there is only standard map. By teaching people how to create maps to reflect their lived environments, as you've mentioned, maps are able to take many shapes and forms. An important aspect of this is also recognizing that maps produced by certain institutions are driven by either social or political agendas that do not necessarily reflect those of the people. As a result, we have to acknowledge the harm that map-making tools and GIS can have on communities and focus on giving the power back to the community. A great example of giving power back to the community that you've mentioned is the LA Community Fridges map, which demonstrates how people are able to create a map and use it to help those in their communities. This also emphasizes the importance of having open-source mapping tools, so that people can work together, find ways to help alleviate those social issues, and bring to light different perspectives.
